### PR TITLE
Feat: Improve consultation request

### PR DESCRIPTION
**Pull request by maulanayuseph**
**Task**
-
[[Web] [Revisi redaksi di halaman Selamat Datang untuk Telekonsultasi Dokter]](https://airtable.com/appMJE57FaKQcEUwA/tbl6no9dhW3sDRJhb/viwR9haftxWJ4p6ty/rechkCkdAYGEyJ7ao?blocks=hide)
[[Web] [Revisi Redaksi pada tombol "Tanyakan Sekarang" diganti dengan "Daftar Konsultasi" pada halaman Isoman Pikobar]](https://airtable.com/appMJE57FaKQcEUwA/tbl6no9dhW3sDRJhb/viwR9haftxWJ4p6ty/recqp7Vulc04q5RzV?blocks=hide)
[[Web] [Penggantian url pada tombol "Tanyakan Sekarang" menjadi: https://pikobar.jabarprov.go.id/consultation-request pada halaman Isoman Pikobar]](https://airtable.com/appMJE57FaKQcEUwA/tbl6no9dhW3sDRJhb/viwR9haftxWJ4p6ty/recGdpkHv3YZSQZQj?blocks=hide)

**Changes**
-
- Change prompt `button` consultation request
- Change `URL` consultation request
- Moved redaction to computed
- Change `URL` hotline isoman pikobar

**Preview**

**Evidence**
title: Improve consultation request
project: Pikobar Website
participants: @maulanayuseph @adzharamrullah @bangunbagustapa @naufalihsank @bangunbagustapa @doohanas

### CODE REVIEW

**Review by adzharamrullah — APPROVED** [review]

**Review by bangunbagustapa — APPROVED** [review]

File: components/_pages/index/IsolasiMandiri/backlinks.js [hunk 1]
```diff
@@ -17,5 +19,7 @@ export {
   infoConsultation,
   infoVitamin,
   detailConsultation,
-  detailVitamin
+  detailVitamin,
+  redaksiKonsultasi,
+  redaksiVitamin
```
Comment (naufalihsank) [inline components/_pages/index/IsolasiMandiri/backlinks.js:24]: suggestion, be consistent with the variable naming (in this case language) with the previous one

**Review by Ibwedagama — APPROVED** [review]

**Review by doohanas — APPROVED** [review]

File: pages/consultation-request/index.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <WelcomePage v-if="showStep === 0" :step.sync="showStep" />
+    <WelcomePage v-if="showStep === 0" :step.sync="showStep" :consultation="true" />
```
Comment (naufalihsank) [inline pages/consultation-request/index.vue:3]: minor, isn't it enough to write the props `consultation` for `true` value like this ? 🤔
```
<WelcomePage v-if="showStep === 0" :step.sync="showStep" consultation />
```

File: components/_pages/index/IsolasiMandiri/backlinks.js (hunk 1 above)
Comment (maulanayuseph) [reply components/_pages/index/IsolasiMandiri/backlinks.js]: Thanks for suggest 👍

File: pages/consultation-request/index.vue (hunk 2 above)
Comment (maulanayuseph) [reply pages/consultation-request/index.vue]: You right, can sent with a default value `true` 😄, fixed on this commit [#464739c](https://github.com/jabardigitalservice/pikobar-jabarprov-go-id/pull/456/commits/464739cf7543cec13cf4b28f474d02a1e6089035)

**Review by naufalihsank — APPROVED** [review]

**Pull request merged**